service_capacity_modeling/capacity_planner.py

```diff
@@ -365,16 +374,17 @@ def _plan_certain(
 
         plans = []
         for instance in hardware.instances.values():
+            exact_match = instance.family_separator in instance.family
```

Comment:
This will always return true false. I think you want to do the for loop like I suggested inside the method against the strings in instance_families

Reply:
@jolynch - Does it make sense to have separate functions? 
def _allow_instance
and
def _allow_drives

We use this function for the disk drives below on line 377. Should we keep this change specific to instance?
https://github.com/Netflix-Skunkworks/service-capacity-modeling/blob/main/service_capacity_modeling/capacity_planner.py#L377